packages/opentelemetry-cloud-trace-exporter/test/exporter.test.ts

```diff
@@ -42,6 +54,20 @@ describe('Google Cloud Trace Exporter', () => {
         assert.deepStrictEqual(id, 'not-real');
       });
     });
+
+    it('should construct exporter in GCE environment without args', async () => {
```

Comment:
s/GCE/GCP/ - the metadata server (even though it's called the GCE Metadata server) is available in all/most GCP compute products.